nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadService.java

```diff
@@ -106,6 +121,28 @@ public long getLength() {
     String getDownloadUrl(String repositoryName, String docId, String xpath, String filename);
 
     /**
+     * Gets the URL to use to download the blobs identified by a storage key.
+     *
+     * <p>
+     * The URL is relative to the Nuxeo Web Application context.
+     * <p>
+     *
+     * @param blobs The key of stored blobs to download
```

Comment:
Incorrect param name, it's `key` in your code and I think it should be named `storeKey` to be more precise and link with the `store` method.
No uppercase for the initial word after the parameter (or after the `@return` below).